administrator/components/com_joomlaupdate/src/Model/NotificationModel.php

```diff
@@ -82,16 +82,45 @@ public function sendNotification($type, $oldVersion, $newVersion): void
             'url'        => Uri::root(),
         ];
 
+        // Determine the default admin language and load the language file for the fallback and default language
+        $defaultLocale   = ComponentHelper::getParams('com_languages')->get('administrator', 'en-GB');
+        $defaultLanguage = $app->getLanguage();
+        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, 'en-GB', true, true);
+        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $defaultLocale);
+
         // Send emails to all receivers
         foreach ($emailReceivers as $receiver) {
-            $params = new Registry($receiver->params);
-            $jLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, 'en-GB', true, true);
-            $jLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $params->get('admin_language', null), true, true);
+            $receiverParams   = new Registry($receiver->params);
+            $receiverLocale   = $receiverParams->get('admin_language', $defaultLocale);
```

Comment:
```suggestion
            $receiverParams = new Registry($receiver->params);
            $receiverLocale = $receiverParams->get('admin_language', $defaultLocale);
```
PHPCS (my mistake from previous suggestion).